src/app/constants.ts

```diff
@@ -67,3 +67,7 @@ export const MESSAGE_FLAGS = {
     FLAG_PUBLIC_KEY: 131072, // whether to attach the public key
     FLAG_SIGN: 262144 // whether to sign the message
 };
+
+export const ENCRYPTED_STATUS = {
```

Comment:
Probably enum is better as you can use it in Attachment interface as a type instead of generic number.

Reply:
Yep, you're right. I have to convert all over time.